Assets/Tests/InputSystem/Plugins/InputForUITests.cs

```diff
@@ -140,6 +144,37 @@ public void SendWheelEvent()
         Assert.That(m_InputForUIEvents[0].asPointerEvent.scroll, Is.EqualTo(new Vector2(0, 1)));
     }
 
+    [Test]
+    [Category("InputForUI")]
+    [TestCase(true)]
```

Comment:
I have this one failing locally right now.

Reply:
Have been discussing with @Pauliusd01 and @ekcoh - There's an issue with the way input tests are setup which makes their behaviour inconsistent, which needs addressing as a separate issue. For example, running the whole of the "Tests.dll" playmode section makes this test pass, but running just the sub-group "InputForUITests" has this test - and on another run, the rest in that group - fail. I don't think it's worth holding this PR up, as all manual testing shows the change is working as intended.